src/components/LoginCard/index.jsx

```diff
@@ -1,50 +1,56 @@
 import React, { useState } from 'react';
 import { Link, useHistory } from 'react-router-dom';
 
-import { Button, Card, Form, Input, Space, Typography } from 'antd';
+import { Button, Card, Form, Input, Modal, Space, Typography } from 'antd';
 
 import { GoogleCircleFilled, LockOutlined, UserOutlined } from '@ant-design/icons';
 
-import './LoginCard.scss';
-import { useAuth } from 'services/AuthProvider';
+import { useAuth } from 'hooks/Hooks';
 import { googleProvider } from 'services/FirebaseService';
+import routes from 'constants/routes';
+
+import './styles.scss';
 
 const { Text } = Typography;
 
 function LoginCard() {
   const { emailLogin, googleLogin } = useAuth();
-  // eslint-disable-next-line no-unused-vars
-  const [loading, setLoading] = useState(false);
+  const [visible, setVisible] = useState(false);
+  const [modalText, setModalText] = useState('');
   const history = useHistory();
 
+  const promptModal = (message) => {
```

Comment:
lets use `function()` instead of arrow functions. this is to follow the Stepdown Rule where the function should always reside above the callee function. using arrow functions  allows us to bypasses that rule. think of arrow functions as `var` variables and `function()`s as `let`.

more on that here:

https://dzone.com/articles/the-stepdown-rule
https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var